sound/soc/sof/sof-of-dev.c

```diff
@@ -31,6 +31,35 @@ const struct dev_pm_ops sof_of_pm = {
 };
 EXPORT_SYMBOL(sof_of_pm);
 
+struct snd_sof_of_mach *sof_of_machine_select(struct snd_sof_dev *sdev)
+{
+	struct snd_sof_pdata *sof_pdata = sdev->pdata;
+	const struct sof_dev_desc *desc = sof_pdata->desc;
+	struct snd_sof_of_mach *mach = desc->of_machines;
+
+	if (!mach)
+		return NULL;
+
+	for (; mach->board; mach++) {
+		int score = of_machine_is_compatible(mach->board);
```

Comment:
The 'score' looks like for debug purpose. May be we can change it to following?

if (of_machine_is_compatible(mach->board)) {
 // ...
}